src/sentry/models/outbox.py

```diff
@@ -537,23 +537,37 @@ def process_shard(
     @contextlib.contextmanager
     def process_coalesced(self) -> Generator[OutboxBase | None, None, None]:
         coalesced: OutboxBase | None = self.select_coalesced_messages().last()
+        first_coalesced: OutboxBase | None = self.select_coalesced_messages().first() or coalesced
+        tags = {"category": "None"}
+
+        if coalesced is not None:
+            tags["category"] = OutboxCategory(self.category).name
+            assert first_coalesced
+            metrics.timing(
+                "outbox.coalesced_net_queue_time",
+                datetime.datetime.now().timestamp() - first_coalesced.date_added.timestamp(),
```

Comment:
Will both these datetimes be in the same timezone? `datetime.datetime` is a naive datetime (implicit system timezone). The `date_added` column will be in UTC as its default value is `timezone.now`

Reply:
Ahh that's a good point, I'll convert the `now` timestamp to UTC first just to make sure this is the case